README.md

```diff
@@ -1,277 +1,16 @@
-# Keycloak
+# Keycloak Node.js Adapter
 
 [![Build Status](https://travis-ci.org/keycloak/keycloak-nodejs-connect.svg?branch=master)](https://travis-ci.org/keycloak/keycloak-nodejs-connect)
 [![Dependency Status](https://img.shields.io/david/keycloak/keycloak-nodejs-connect.svg?style=flat-square)](https://david-dm.org/keycloak/keycloak-nodejs-connect)
 
-[Keycloak](http://keycloak.jboss.org/) is a standalone authentication
+[Keycloak](http://keycloak.org/) is a standalone authentication
 server, akin to a standalone database.  It provides hooks for federated
 authentication, including authenticating against various social networks
 and OAuth providers (G+, Facebook, etc).
 
 This module makes it simple to implement a Node.js Connect-friendly
-application that uses Keycloak for its authentication and authorization needs.
+application that uses Keycloak for its authentication and authorization needs. For usage instructions, please read the [documentation](https://keycloak.gitbooks.io/securing-client-applications-guide/content/v/2.2/topics/oidc/oidc-overview.html).
 
```

Comment:
The link here points to the official KC documentation. But that documentation does not include what has been removed from this README. What's there is info about the browser based KC adapter. Shouldn't all of this documentation below be moved to the gitbooks location?